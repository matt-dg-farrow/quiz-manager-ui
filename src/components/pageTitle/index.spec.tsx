import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import PageTitle from ".";

describe("The PageTitle component", () => {
  const wrapper = shallow(<PageTitle text="Title" />);

  it("should contain the correct number of components", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
