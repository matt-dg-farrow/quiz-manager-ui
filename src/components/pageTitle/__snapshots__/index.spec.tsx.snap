// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The PageTitle component should contain the correct number of components 1`] = `
<div
  className="quiz-manager-page-title-text"
>
  Title
</div>
`;
